_specs_/websocket/schemas/crypto/actions/l2Book.yml

```
@@ -0,0 +1,26 @@
+SubscribeAction:
+  allOf:
+    - $ref: '../../index.yml#/SubscribeAction'
+    - properties:
+        params:
+          pattern: /^XL2\.((?<from>([A-Z]{3})-(?<to>[A-Z]{3}))|\*)$/
```

Same comment as above
```suggestion
          pattern: /^XL2\.((?<from>([A-Z]*)-(?<to>[A-Z]{3}))|\*)$/
```